ui/app/AppLayouts/Wallet/popups/swap/SwapInputParamsForm.qml

```diff
@@ -13,10 +13,13 @@ QtObject {
     property int selectedNetworkChainId: -1
     property string fromTokensKey: ""
     property string fromTokenAmount: ""
-    property string toTokenKey: ""
+    property string toTokenKey: root.defaultToTokenKey
     property string toTokenAmount: ""
     property double selectedSlippage: 0.5
 
+    // default token key
+    property string defaultToTokenKey: ""
```

Comment:
this has been added as it will be needed for the network selection flow that is yet to be finalised. The flow would be that if the selected token is not present on new network selected, the form should be reset. And this will be needed in that case. Added in advance here in this PR.